server/schedule/operator/operator.go

```diff
@@ -76,9 +75,15 @@ func NewOperator(desc, brief string, regionID uint64, regionEpoch *metapb.Region
 		level:           level,
 		AdditionalInfos: make(map[string]string),
 		ApproximateSize: approximateSize,
+		timeout:         time.Duration(maxDuration) * time.Second,
 	}
 }
 
+// Sync some attribute with the given timeout.
```

Comment:
Maybe we can use option to specify which fields to sync

Reply:
In past, only timeout needs to sync. It should be done if sync one more properties